src/frontends/tensorflow/tests/convert_unsupported.cpp

```diff
@@ -75,6 +78,14 @@ shared_ptr<Model> convert_model_partially(const string& model_path) {
 
     return model;
 }
+
+ov::OutputVector incorrect_less_translator(const ov::frontend::NodeContext& node) {
+    // NOTE: pay attention that this is a fake translator for Less operation
+    // only serves for testing purposes
+    TENSORFLOW_OP_VALIDATION(node, false, "Less expects ten inputs.");
+    return {};
+}
```

Comment:
To test both known and unknown exception handlers specialize this function by the exception statement. I would recommend to implement translator itself as a lambda function in the generation function that accepts another function as a parameter. That function will be used to generate the exception. Doing so you can build "incorrect" translator with configured exception type.

Reply:
Discussed, agreed that not needed.